tests/unittest/_torch/modules/tests_lora_modules/test_lora_mlp_pytorch_flow_vs_trt.py

```diff
@@ -0,0 +1,283 @@
+import unittest
```

Comment:
I don't think we need to use trt as reference all the time. We can use torch as reference directly to prevent the overhead of building engine, the complicated codes of initializing the TRT engine and inputs.